### PR TITLE
fix(ci): clear disk space for scanner vulns

### PR DESCRIPTION
**Pull request by RTann**
### Description

Update Scanner dev vuln update pipeline to clear disk space. I also updated the release pipeline to call the job preamble

### User-facing documentation

- [x] CHANGELOG is updated **OR** update is not needed
- [x] [documentation PR](https://spaces.redhat.com/display/StackRox/Submitting+a+User+Documentation+Pull+Request) is created and is linked above **OR** is not needed

Not needed

### Testing and quality

- [x] the change is production ready: the change is GA or otherwise the functionality is gated by a feature flag
- [x] CI results are inspected

This IS the test

#### Automated testing

- [x] modified existing tests

#### How I validated my change

This IS the test

### CODE REVIEW

File: .github/workflows/scanner-dev-vuln-update.yaml [hunk 1]
```diff
@@ -107,6 +108,10 @@ jobs:
         fetch-depth: 0
         ref: ${{ github.event.pull_request.head.sha || github.ref }}
 
+    - uses: ./.github/actions/job-preamble
+      with:
+        gcp-account: ${{ secrets.GCP_SERVICE_ACCOUNT_STACKROX_CI }}
+
```
Comment (dcaravel) [inline .github/workflows/scanner-dev-vuln-update.yaml:114]: The cleanup (`docker system prune...`) in the preamble wasn't enough for the release job, also had to also add: 
https://github.com/stackrox/stackrox/blob/a0c7efe935ef8ac0d4cc051402f68e16fe19f4e8/.github/workflows/scanner-release-vuln-update.yaml#L109-L111

May make sense to add that here as well (or to the preamble job itself if will not break other workflows)

Comment (RTann) [reply .github/workflows/scanner-dev-vuln-update.yaml:130]: I updated this job to include that, but I also modified the release CI to call the preamble

**Review by dcaravel — APPROVED** [review]
Assuming CI passes, looks good to me.

File: .github/workflows/scanner-dev-vuln-update.yaml [hunk 2]
```diff
@@ -95,6 +95,13 @@ jobs:
     - build-updater
     - download-nvd
     runs-on: ubuntu-latest
+    container:
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.69
+      volumes:
+      # The updater makes heavy use of /tmp files.
+      - /tmp:/tmp
+      - /usr:/mnt/usr
+      - /opt:/mnt/opt
```
[the remark this answers is not in the document]
Comment (RTann) [reply .github/workflows/scanner-dev-vuln-update.yaml:104]: You're right. I originally added this to guarantee a Go version, but that was when I accidentally put this in the `build-updater` step. Once I moved it to the `run-updater` step, this was no longer needed

File: .github/workflows/scanner-dev-vuln-update.yaml [hunk 3]
```diff
@@ -107,6 +107,21 @@ jobs:
...
+        gcp-account: ${{ secrets.GCP_SERVICE_ACCOUNT_STACKROX_CI }}
+
+    - name: Free up disk space
+      shell: bash
+      run: |
+        set +e
+        set -x
+        df -h
+        for delete in /usr/share/dotnet /usr/local/lib/android /opt/ghc /opt/hostedtoolcache/CodeQL; do
+          rm -rf "/mnt${delete:?}"
```
Comment (dcaravel) [inline .github/workflows/scanner-dev-vuln-update.yaml:121]:
```suggestion
          rm -rf "${delete:?}"
```

**Review by dcaravel — CHANGES_REQUESTED** [review]

**Comment by RTann** [comment]
Keep failing style checks due to out-of-space (ironic). See https://github.com/stackrox/stackrox/pull/12290

**Comment by dcaravel** [comment]
> Keep failing style checks due to out-of-space (ironic). See #12290

`dev` workflow now looks to be passing consistently now however, progress!

**Review by dcaravel — APPROVED** [review]
LGTM

**Pull request merged**